dask/array/core.py

```diff
@@ -2170,6 +2170,12 @@ def from_zarr(url, component=None, storage_options=None, chunks=None, **kwargs):
         Passed to ``da.from_array``, allows setting the chunks on
         initialisation, if the chunking scheme in the on-disc dataset is not
         optimal for the calculations to follow.
+    name : str, optional
+        The key name to use for the array. Defaults to a hash of ``x``.
```

Comment:
This description can be shortened and references to `x` (from the `from_array` docstring) should be removed. @mpeaton what do you think about something like "The key name to use for the array. If not specified a unique name will generated."?